a2ml/api/utils/file_uploader.py

```diff
@@ -0,0 +1,58 @@
+import boto3
+import os
+import sys
+import threading
+
+from boto3.s3.transfer import TransferConfig
+from uuid import uuid4
+
+class FileUploader(object):
```

Comment:
@holyketzer are there plans to support delete as well? When I delete the dataset it does remove it from the provider, but it remains in minio.

Reply:
hm. in case of upload it should storage file forever as I understand?
or after provider handled and cache file we can remove it? In this case we can do it immediately after the command is done.